repl/src/main/scala/ammonite/repl/Util.scala

```diff
@@ -172,6 +172,15 @@ object Parsers {
     }
   }
 
+  val CompilationUnit = P( WL ~ Semis.? ~ (!"@\n" ~ (scalaparse.Scala.Import | Prelude ~ BlockDef | StatCtx.Expr)).!.rep(sep=Semis) ~ Semis.? ~ WL )
```

Comment:
Could we factor out the common bits of `CompilationUnit` here and `Splitter` above into a shared value?